internal/pkg/otel/configtranslate/otelconfig.go

```diff
@@ -380,6 +380,7 @@ func translateEsOutputToExporter(cfg *config.C) (map[string]any, error) {
 		return nil, err
 	}
 	// we want to use dynamic indexing
+	esConfig["logs_index"] = "" // needs to be empty for logs_dynamic_index
```

Comment:
This is a breaking change in the elasticsearchexporter. In order to use logs_dynamic_index the logs_index must be empty.